app/src/main/java/org/resolvetosavelives/red/newentry/search/PatientAddress.kt

```diff
@@ -12,13 +12,14 @@ data class PatientAddress(
     @PrimaryKey
     val uuid: String,
 
-    val colonyOrVillage: String?,
+    val colonyOrVillage: String,
 
-    val district: String?,
+    val district: String,
 
-    val state: String?,
+    val state: String,
 
-    val country: String?,
+    //todo: Don't use India as the country for everyone!
```

Comment:
When did you start writing single line comments without a space? and without sentence case?